### DISCUSSION TITLE
MIUI13 and connection to my BMW

### DISCUSSION THREAD

**Discussion by Plastued** [BimmerGestalt/AAIdrive#657] [Q&A]
Hi there, today my redmi note 9 pro made an update to MIUI13 and AAIdrive couldn't connect via the myBMW app..
The myBMW app tells me that my phone is connected with the car, but it seems that the AAIdrive app couldn't get the connection.. Does someone know a solution?! It would be very sad when i can't use AAIdrive anymore, it is a very great app!

**Comment by hufman** [comment]
When you say MyBMW connects to the car, is the Apps connection actually running? For example, the MINI app looks like this when the Apps connection is running properly:
![image](https://user-images.githubusercontent.com/1592375/212799109-fd2a0948-d851-426b-8fec-13a84a1afe3e.png)![image](https://user-images.githubusercontent.com/1592375/212799142-fa29c1c7-6c24-494c-8f59-41d5f03b224d.png)

![image](https://user-images.githubusercontent.com/1592375/212798999-f1e5a75c-a3eb-4bc8-bc72-a18ea957f8d2.png)![image](https://user-images.githubusercontent.com/1592375/212799034-d1699e89-9d3a-4108-aae5-b37b1ba00397.png)

Once MyBMW is connected, AAIdrive should find the port and connect through it.

**Reply by Plastued** [reply]
Thank you for your answer! MyBMW is just telling me that the vehicle is detected (Bluetooth connection is established)
Okay, so my problem is with the myBMW app.. I'd try later if i can get a connection!

**Reply by hufman** [reply]
You could also try downgrading to MyBMW 2.9, it seems 2.10 and newer have some connection bugs on new Android versions (discussed in #638).
